packages/core/content-releases/admin/src/pages/Releases.tsx

```diff
@@ -1,22 +1,56 @@
-import { Main } from '@strapi/design-system';
-import { Route, Switch } from 'react-router-dom';
+/*
+ *
+ * Releases
+ *
+ */
+import React from 'react';
 
-import { pluginId } from '../pluginId';
+import { Button, Main, HeaderLayout } from '@strapi/design-system';
+import { Plus } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+
+import { AddReleaseDialog } from '../components/AddReleaseDialog';
+
+const ReleasesPage = () => {
+  const [showAddReleaseDialog, setShowAddReleaseDialog] = React.useState(false);
+  const { formatMessage } = useIntl();
+
+  const total = 0; // TODO: replace it with the total number of releases
+
+  const subtitle = formatMessage(
+    {
+      id: 'content-releases.pages.Releases.header-subtitle',
+      defaultMessage: '{number, plural, =0 {No releases} one {# release} other {# releases}}',
+    },
+    { number: total }
+  );
+
+  const toggleAddReleaseDialog = () => {
+    setShowAddReleaseDialog((prev: boolean) => !prev);
```

Comment:
I don't think `:bool` should be necessary here 🤔  isn't typescript able to infer the type? Or maybe if you declare it with the generic? `React.useState<boolean>(false)`

Reply:
yes it is not necessary